### PR TITLE
Chore/profile view model

### PR DESCRIPTION
**Pull request by sayaandreas**

### CODE REVIEW

File: app/src/main/java/com/example/tsaving/webservice/WebServices.kt [hunk 1]
```diff
@@ -49,10 +50,10 @@ interface WebServices {
     suspend fun verifyAccount(@Body body: VerifyRequestModel): GenericResponseModel<EmailResponse>
 
     @GET(VIEW_PROFILE)
-    suspend fun viewProfile()
+    suspend fun viewProfile(@Header("Authorization") token: String) : ProfileResponseModel
 
     @PUT(UPDATE_PROFILE)
-    suspend fun updateProfile()
+    suspend fun updateProfile(@Header("Authorization") token: String, @Body body: EditProfileRequestModel) : EditProfileResponseModel
```
Comment (david1312) [inline app/src/main/java/com/example/tsaving/webservice/WebServices.kt]: maybe can use generic

File: app/src/main/java/com/example/tsaving/webservice/WebServices.kt [hunk 2]
```diff
@@ -49,10 +50,10 @@ interface WebServices {
     suspend fun verifyAccount(@Body body: VerifyRequestModel): GenericResponseModel<EmailResponse>
 
     @GET(VIEW_PROFILE)
-    suspend fun viewProfile()
+    suspend fun viewProfile(@Header("Authorization") token: String) : ProfileResponseModel
```
Comment (david1312) [inline app/src/main/java/com/example/tsaving/webservice/WebServices.kt]: maybe can use generic class

**Review by david1312 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/tsaving/webservice/WebServices.kt (hunk 2 above)
Comment (sayaandreas) [reply app/src/main/java/com/example/tsaving/webservice/WebServices.kt]: done vid

File: app/src/main/java/com/example/tsaving/webservice/WebServices.kt (hunk 1 above)
Comment (sayaandreas) [reply app/src/main/java/com/example/tsaving/webservice/WebServices.kt]: done

**Review by david1312 — APPROVED** [review]
ok

**Pull request merged**